main.py

```diff
@@ -37,9 +39,23 @@ def main():
             print("Cleaning up namespace %s" % namespace.metadata.name)
             # real cleanup will be enable later once it's tested
             v1api.delete_namespace(namespace.metadata.name, client.V1DeleteOptions())
+            cleaned += 1
```

Comment:
You can also add how many namespaces were scanned. May be interesting at some point. NTH.

Reply:
ohh me like. on it.